src/ray/raylet/node_manager.cc

```diff
@@ -575,13 +587,12 @@ void NodeManager::HandleReleaseUnusedBundles(
   }
 
   for (const auto &worker : workers_associated_with_unused_bundles) {
-    RAY_LOG(DEBUG)
-        << "Destroying worker since its bundle was unused. Placement group id: "
-        << worker->GetBundleId().first
-        << ", bundle index: " << worker->GetBundleId().second
-        << ", task id: " << worker->GetAssignedTaskId()
-        << ", actor id: " << worker->GetActorId()
-        << ", worker id: " << worker->WorkerId();
+    RAY_LOG(INFO) << "Destroying worker since its bundle was unused. Placement group id: "
```

Comment:
It is not clear to me what it means by `since its bundle was unused`. Can you explain in more high level?